src/Form/DataTransformer/ModelToIdPropertyTransformer.php

```diff
@@ -59,10 +59,10 @@ class ModelToIdPropertyTransformer implements DataTransformerInterface
     protected $toStringCallback;
 
     /**
-     * @param string        $className
-     * @param string        $property
-     * @param bool          $multiple
-     * @param callable|null $toStringCallback
+     * @param string          $className
+     * @param string|string[] $property
+     * @param bool            $multiple
+     * @param callable|null   $toStringCallback
      *
      * @phpstan-param class-string<T> $className
      * @phpstan-param null|callable(object, string): string $toStringCallback
```

Comment:
Now the callable can accept `string[]` no ?

SO it should be
```
@phpstan-template P of string|string[]
@phpstan-param class-string<T> $className
@phpstan-param null|callable(object, P): string $toStringCallback
```

but I'm not sure

Reply:
```
 PHPDoc tag @template P for method                                      
         Sonata\AdminBundle\Form\DataTransformer\ModelToIdPropertyTransformer:  
         :__construct() with bound type array<string>|string is not supported.

Reply:
@VincentLanglet this seems to work?

`@phpstan-param null|callable(object, string|string[]): string $toStringCallback`

Reply:
did you tried @phpstan-template P ?

Reply:
What would be the benefit of that? What is `P` then?

Reply:
If you write `callable(object, string|string[])`

passing a `callable(object, string)` would be invalid.
but it should be valid if the property is a string.

Reply:
But we don't define `P` anywhere? So how does this help?

Anyway I changed it.